keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -94,6 +96,14 @@ As part of this proposal, we are mainly proposing three changes:
 
 1. Relax API validation on PVC update so as reducing `pvc.Spec.Resoures` is allowed, as long as requested size is `>pvc.Status.Capacity`.
 2. Add a new field in PVC API called - `pvc.Status.AllocatedResources` for the purpose of tracking used storage size. By default only api-server or resize-controller can set this field.
+3. Add a new field in PVC API called - `pvc.Status.ResizeStatus` for purpose of tracking status of volume expansion. Following are possible values of newly introduced `ResizeStatus` field:
+   - nil // General default because type is a pointer.
+   - empty // When expansion is complete, the empty string is set by resize controller or kubelet.
```

Comment:
It can be nil or empty and don't think we need an explicit default of empty. It might help us later down the road when we want to distinguish between values that were never set vs values that just has been cleared off via resize operation.